includes/pet-details-form.php

```diff
@@ -146,12 +146,12 @@
 </p>
 
 <p class="h-spacing-base">
-    <label for="date">Date last seen:</label>
+    <label for="date">Date <span class="seen-found-text">last seen</span>:</label>
```

Comment:
These classes should be `.js-` hooks.  So `.js-seen-found-text`.  Not liking the naming either